.env.example

```
@@ -55,19 +66,25 @@ POSTGRES_PORT=50105
 NUQ_DATABASE_URL=postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@pulse_postgres:5432/${POSTGRES_DB}
 
 # Redis
+# Single Redis instance shared by all services (Firecrawl API, MCP Server, Webhook Bridge)
+# All services use REDIS_URL for consistency - no service-specific URLs needed
 REDIS_PORT=50104
 REDIS_URL=redis://pulse_redis:6379
-REDIS_RATE_LIMIT_URL=redis://pulse_redis:6379
 BULL_AUTH_KEY=@
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BULL_AUTH_KEY key should go before the REDIS_PORT key

(UnorderedKey)